.github/workflows/test-webassembly.yml

```diff
@@ -26,7 +26,6 @@ env:
   WASM_BINDGEN_TEST_ONLY_WEB: 1
   WASM_BINDGEN_SPLIT_LINKED_MODULES: 1
   RSTEST_TIMEOUT: 90
-  GECKODRIVER: "${{ env.GECKOWEBDRIVER }}"
```

Comment:
one of the annoying things about actions is that a "lint error" will silently fail making the rest of the workflow green, which is why the last #2832 pr got merged. Apparently env can't be used here b/c its a global which creates a circular reference. bindgen needs the driver to be under `GECKODRIVER` and not geckowebdriver like the warp builder provides